haptools/sim_phenotype.py

```diff
@@ -301,6 +376,13 @@ def simulate_pt(
     else:
         log.info("Loading genotypes from VCF/BCF file")
         gt = Genotypes(fname=genotypes, log=log)
+
+    tr_gt = None
+    if repeats:
+        log.info("Loading TR genotypes")
+        tr_gt = GenotypesTR(fname=repeats, log=log)
+        tr_gt.read(region=region, samples=samples, variants=haplotype_ids)
```

Comment:
We could probably use our new `type_ids` superpowers here
```suggestion
        tr_gt.read(region=region, samples=samples, variants=hp.type_ids["R"])
```